### PR TITLE
imperial ministry and acadamy

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/han/academy.xml [hunk 1]
```diff
@@ -33,6 +33,9 @@
     <Requirements>
       <Techs datatype="tokens">-phase_village phase_city</Techs>
     </Requirements>
+    <History>
+        The Imperial academy or, Taixue, was the highest place of study in ancient imperial china. The academy taught students Confucianism and literature as well as preparing them to take jobs in the empire’s government. It was a large school, at its peak having over 30,000 students and facalty.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/han/academy.xml:37]:
```suggestion
        The Imperial Academy, or Taixue, was the highest place of study in ancient imperial China. The academy taught students Confucianism and literature as well as preparing them to take jobs in the empire’s government. It was a large school, at its peak having over 30,000 students and faculty.
```

File: 0-ad-history-mod-package/simulation/templates/structures/han/ministry.xml [hunk 2]
```diff
@@ -42,6 +42,9 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        The Han dynasty had a large bureaucratic force to help efficiently manage the empire’s vast resources. The bureaucrats were organized into nine courts, each put under the charge of an imperial minister who was in charge of a specific aspect of the government.
```
Comment (indoptogopt) [inline 0-ad-history-mod-package/simulation/templates/structures/han/ministry.xml:46]:
```suggestion
        The Han dynasty had a large bureaucratic force to help efficiently manage the empire’s vast resources. The bureaucrats were organized into nine courts, each put under the charge of an imperial minister who was responsible of a specific aspect of the government.
```

**Review by indoptogopt — CHANGES_REQUESTED** [review]

File: 0-ad-history-mod-package/simulation/templates/structures/han/ministry.xml (hunk 2 above)
Comment (indoptogopt) [reply 0-ad-history-mod-package/simulation/templates/structures/han/ministry.xml:46]: I would remove one the "charges"

**Comment by TheShadowOfHassen** [comment]
Can I merge this?

**Comment by indoptogopt** [comment]
Yes.

**Pull request merged**